index.rst

```diff
@@ -15,5 +10,265 @@ Change Record
+
+* Level 4 denotes internal milestones within a team.
+
+*O(few hundred)* milestones have been defined to-date to track the progress of DM activities.
+
+Relationship between milestones, as well as between milestones and planning packages are captured:
+milestones typically *block* planning packages, milestones can also *relate* to other milestones.
+
+Baseline Long-term Plan
+-----------------------
+
+Planning packages, together with Milestones form the *Baseline Plan*. The plan is under change control.
+
+The plan is stable, any change to the plan, including changing scope, cost or schedule must be approved
+by the appropriate change control body:
+
+* Change Control Board approval for any non-trivial change to a planning package or Level 1 and
+  Level 2 milestone
+
+* DM's TCT approval for Level 3 milestones
+
+* no approval is needed for Level 4 milestones, however any non-trivial change need to be communicated
+  to the DM team
+
+* trivial changes, such as small corrections / clarifications to narrative that do not affect
+  scope, time or budget are allowed without approval.
+
+Typically, adjustments to Level 1, 2 and 3 milestones are made every 6 or 12 months. Changes to Level 4
+milestones can occur more frequently.
+
+Short-term Planning
+-------------------
+
+Short-term Plan is managed through *epics* and *stories*, and executed through *cycles*.
+
+In the EVMS world, epics map directly to *activities*, and stories map to *activity steps*.
+
+Epics and Stories
+~~~~~~~~~~~~~~~~~
+
+Typically, planning packages encompass relatively large blocks of work. To plan and execute work
+in details, finer-grain planning is required. This is implemented through epics and stories.
+
+Each epic captures a non-trivial work that is associated with a subset of work defined in a
+planning package. Each epic has a clear description, well defined deliverables, and
+relationships with a planning package (an epic *implements* a PP), and a Milestone (an epic
+*blocks* a milestone). Epics are resources loaded, and have start and end dates defined.
+
+Each epic is broken into smaller chunks of work, called *Stories*. Stories are primarily used
+to define and manage short-term activities of individual developers.
+
+Epics and stories are used to track all software work, as well as work on delivering hardware and
+services that are tightly coupled with software (example: "Deliver OpenStack-based Test Cluster
+running RedHat 6.1".) Details of work on standalone activities do not have to be captured on daily
+bases through stories (example: "Deliver 10Gbit link between Chile and USA")
+
+.. _cycles-and-releases:
+
+Cycles and Releases
+~~~~~~~~~~~~~~~~~~~
+
+The work is executed through 6-month *cycles*. Scope of work for each cycle is driven by the Baseline
+Plan, in particular by milestones. Each cycle ends with a new software release. Releases are
+thoroughly tested and documented.
```

Comment:
We've previously discussed the timing of releases within the cycle (ie, freeze two weeks before the cycle close, release promptly at the cycle change). If that is now official policy, it should be documented -- this document probably isn't the place, but I'm not sure where is. DMTN-020? The developer guide? Somewhere else?